apps/prairielearn/src/tests/activeAccessRestriction.test.ts

```diff
@@ -39,10 +41,22 @@ describe(
 
     beforeAll(async function () {
       await helperServer.before()();
-      context.examId = await sqldb.queryRow(sql.select_exam11, IdSchema);
+      const { id: course_instance_id } = await selectCourseInstanceByShortName({
+        course_id: '1',
+        short_name: 'Sp15',
+      });
+      const { id: examId } = await selectAssessmentByTid({
+        tid: 'exam11-activeAccessRestriction',
+        course_instance_id,
+      });
+      const { id: hwId } = await selectAssessmentByTid({
+        tid: 'hw8-activeAccessRestriction',
+        course_instance_id,
+      });
+      context.examId = examId;
       context.examUrl = `${context.courseInstanceBaseUrl}/assessment/${context.examId}/`;
```

Comment:
It seems `courseInstanceBaseUrl` is also based on ID 1.

Reply:
Changed in e70b0daf96c0a991b5cc736e9f596e2e6b94076c.